docs(dns): add docs for dns&domain product (#909)
scaleway/docs-content: network/dns-cloud/how-to/configure-autorenew.mdx

---
meta:
  title: How to configure autorenew for an internal domain name
  description: This page explains how to configure autorenew for an internal domain name with Scaleway Domains & DNS.
content:
  h1: How to configure autorenew for an internal domain name
  paragraph: This page explains how to configure autorenew for an internal domain name with Scaleway Domains & DNS.
dates:
  validation: 2022-10-12
  posted: 2022-10-12
---

Autorenew is a feature that allows you to automatically renew the validity period of a domain name registered with Scaleway Domains & DNS on a yearly basis.

<Message type="requirement">

- You have an account and are logged into the [Scaleway console](https://console.scaleway.com) 
- You have a domain name registered with Scaleway Domains & DNS

</Message>

## How to activate autorenew

1. Click **Domains & DNS** in the **Network** section of the [Scaleway Console](https://console.scaleway.com) side menu. 

> The Domain & DNS dashboard displays. 

Reviewer: ```suggestion
The Domain & DNS dashboard displays with a list of your internal domains.
```

2. Click the domain name you want to configure.

> 3. Click **Activate** next to **Autorenew** in the domain information section.

Reviewer: ```suggestion
3. Click **Activate** under **Autorenew** in the domain information section.
```

    <Lightbox src="scaleway-domain-autorenew.webp" alt="" />

> Autorenew is activated for your domain name.

Reviewer: ```suggestion
You have activated autorenew for your domain name.
```

## How to deactivate autorenew

> 1. Click **Domains & DNS** in the **Network** section of the [Scaleway Console](https://console.scaleway.com) side menu. 

Reviewer: ```suggestion
1. Click **Domains and DNS** in the **Network** section of the [Scaleway Console](https://console.scaleway.com) side menu. 
```

> The Domain & DNS dashboard displays. 

Reviewer: ```suggestion
The Domain & DNS dashboard displays with a list of your internal domains.
```

> The Domain & DNS dashboard displays with a list of your internal domains.

Reviewer: ```suggestion
The Domains and DNS dashboard displays with a list of your internal domains.
```

2. Click the domain name you want to configure.

> 3. Click **Deactivate** next to **Autorenew** in the domain information section.

Reviewer: ```suggestion
3. Click **Deactivate** under **Autorenew** in the domain information section.
```

4. Click **Validate** in the pop-up to confirm. 

> Autorenew is deactivated for your domain name.

Reviewer: ```suggestion
You have deactivated autorenew for your domain name.
```
